app/src/main/AndroidManifest.xml

```diff
@@ -33,6 +36,8 @@
         <meta-data
             android:name="com.google.android.geo.API_KEY"
             android:value="AIzaSyCa3x16MPeqXAaf35RSN8dFd0rBRJwybNo" />
```

Comment:
You shouldn't have the API_KEY write here, because it's not anymore a secret and someone could stole it.
```
<meta-data
            android:name="com.google.android.geo.API_KEY"
            android:value="${MAPS_API_KEY}" />
```
It's should be remplaced something like this and the key stored on a file that is not add to github and also stored inside GittHub Secrets

Reply:
True thanks for the comment ! I Completely forget this ^^'